packages/zoe/test/unitTests/zcf/test-zcfSeat-exit.js

```diff
@@ -45,7 +42,7 @@ test(`zoe - wrongly throw zcfSeat.exit()`, async t => {
   });
 
   // eslint-disable-next-line no-unused-vars
-  const { creatorFacet } = await E(zoe).startInstance(
+  const { _creatorFacet } = await E(zoe).startInstance(
```

Comment:
can the preceding comment ("no-unused-vars") be dropped?

Reply:
Done